src/main/java/org/apache/maven/plugins/jlink/JLinkMojo.java

```diff
@@ -510,6 +522,12 @@ private File createZipArchiveFromImage(File outputDirectory, File outputDirector
             throws MojoExecutionException {
         zipArchiver.addDirectory(outputDirectoryImage);
 
+        // configure for Reproducible Builds based on outputTimestamp value
+        Date lastModified = new MavenArchiver().parseOutputTimestamp(outputTimestamp);
+        if (lastModified != null) {
```

Comment:
Do we also want to add a line similar to this one?
https://github.com/apache/maven-rar-plugin/commit/6ea9c738f089930b5f27b4e7796808412b1886b0/#diff-bb425c248269250f8709b3657272fc04a50a12d316ceb241c1fa64a2c19f9cadR382

Reply:
in jlink, it's a zip, not a jar, then there are no MANIFEST.MF metadata